File: 01-09/Problem_5/Problem5.java

public class Problem5 {

    static boolean isValid(int number) {
        for (int i=20; i > 2; i--) {
            if (number%i != 0) return false;
        }
        return true;
    }

    public static void main(String[] args) {
        int result = 20;
        while (!isValid(result)) result += 20;
        System.out.println(result);
    }
}

File: 01-09/Problem_5/problem5.c
#include <stdio.h>

int isValid(int n) {
    for (int i=20; i > 2; i--) {
        if (n%i != 0) return 0;
    }
    return 1;
}

int main() {
    int result = 20;
    while (!isValid(result)) result += 20;
    printf("%d\n", result);
    return 0;
}